Doppler.ContactPolicies.Api/Startup.cs

```diff
@@ -52,8 +52,13 @@ public void ConfigureServices(IServiceCollection services)
                         Array.Empty<string>()
                     }
                 });
-                c.SwaggerDoc("v1", new OpenApiInfo { Title = "doppler_contact_policies_api", Version = "v1" });
-                if (!string.IsNullOrEmpty(_baseUrl)) { c.AddServer(new OpenApiServer() { Url = _baseUrl }); };
+                c.SwaggerDoc("v1", new OpenApiInfo {Title = "doppler_contact_policies_api", Version = "v1"});
```

Comment:
Just to continue with project naming convention  
```suggestion
                c.SwaggerDoc("v1", new OpenApiInfo {Title = "Doppler.ContactPolicies.Api", Version = "v1"});
```